Source/items.cpp

```diff
@@ -1010,16 +1010,12 @@ int SaveItemPower(const Player &player, Item &item, ItemPower &power)
 	case IPL_ACUNDEAD:
 		item._iDamAcFlags |= ItemSpecialEffectHf::ACAgainstUndead;
 		break;
-	case IPL_MANATOLIFE: {
-		int portion = ((player._pMaxManaBase >> 6) * 50 / 100) << 6;
-		item._iPLMana -= portion;
-		item._iPLHP += portion;
-	} break;
-	case IPL_LIFETOMANA: {
-		int portion = ((player._pMaxHPBase >> 6) * 40 / 100) << 6;
-		item._iPLHP -= portion;
-		item._iPLMana += portion;
-	} break;
+	case IPL_MANATOLIFE:

```

Comment:
Since we skip the hardcoded values when calculating the items for  DevX, why not just hardcode in the values to maintain reverse compatibility?